packages/datadog-plugin-cypress/test/index.spec.js

```diff
@@ -0,0 +1,73 @@
+'use strict'
+const getPort = require('get-port')
+const { expect } = require('chai')
+
+const agent = require('../../dd-trace/test/plugins/agent')
+const plugin = require('../src')
+const appServer = require('./app/app-server')
+const { ORIGIN_KEY } = require('../../dd-trace/src/constants')
+const {
+  TEST_FRAMEWORK,
+  TEST_TYPE,
+  TEST_NAME,
+  TEST_SUITE,
+  TEST_STATUS,
+  CI_APP_ORIGIN
+} = require('../../dd-trace/src/plugins/util/test')
+
+describe('Plugin', () => {
+  let cypressExecutable
+  let appPort
+  let agentListenPort
+  withVersions(plugin, ['cypress'], (version, moduleName) => {
+    beforeEach(() => {
+      return agent.load(['cypress']).then((agentPort) => {
```

Comment:
Is this even needed if there is no automatic instrumentation?

Reply:
we need to spin up a "fake" agent to read traces from. We probably don't need to load any specific instrumentation though